### PR TITLE
Forward port 7 to main

### PR DESCRIPTION
**Pull request by nkoenig**
# ➡️ Forward port

Port gz-sim7 to main. This one is slightly newer to #2011, and includes some recent changes.

Branch comparison: [https://github.com/gazebosim/<REPO>/compare/<TO_BRANCH>...<FROM_BRANCH>](https://github.com/gazebosim/gz-sim/compare/main...gz-sim7)

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

File: tutorials/blender_procedural_datasets.md [hunk 1]
```diff
@@ -155,9 +155,9 @@ blender rock.blend --python-text procedural_dataset_generator.py -- -o sdf_model
 blender woodland.blend --python-text procedural_dataset_generator.py -- -o sdf_models/woodland
 ```
 
-@image html https://github.com/gazebosim/gz-sim/tree/main/tutorials/files/blender_procedural_datasets/demo_blender_rock.gif "Example of generating a dataset of rock SDF models" width=100%
+![Example of generating a dataset of rock SDF models](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/blender_procedural_datasets/demo_blender_rock.gif)
```
Comment (azeey) [inline tutorials/blender_procedural_datasets.md:158]: There's a missed change on line 96. I think it should be 
```
![Instructions in Blender](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/blender_procedural_datasets/blender_instructions.png)
```

**Review by azeey — COMMENTED** [review]
Just a couple of minor comments. Otherwise, LGTM!

**Comment by mjcarroll** [comment]
@azeey I addressed your feedback, PTAL.

**Review by azeey — APPROVED** [review]

**Comment by nkoenig** [comment]
> @azeey I addressed your feedback, PTAL.

Thanks @mjcarroll

**Pull request merged**